scipy/special/boost_special_functions.h

```diff
@@ -957,11 +957,11 @@ nct_cdf_wrap(const Real v, const Real l, const Real x)
 	return NAN;
     }
     if (v <= 0) {
-	sf_error("nctdtr", SF_ERROR_DOMAIN, NULL);
-	return NAN;
+	    sf_error("nctdtr", SF_ERROR_DOMAIN, NULL);
+	    return NAN;
     }
     if (std::isinf(x)) {
-	return  (x > 0) ? 1.0 : 0.0;
+	    return  (x > 0) ? 1.0 : 0.0;
```

Comment:
some kind of editor setting maybe?

Reply:
@dschmitz89.  We don't use 8 space indents in C++ code in special. Can you address this?

Reply:
Done in last commit.